### ISSUE TITLE
[Bug]: processing depth if setting none?

### ISSUE THREAD

**Issue by mykeehu** [Mikubill/sd-webui-controlnet#639]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I have a depth map, but when I load it and select the control_depth model, the generated pattern appears as if it has been processed with inversion color, even though it is set to none.

Here is preview of annotator:
![image](https://user-images.githubusercontent.com/6390413/226426157-ace757a7-d296-4b5b-af76-d8ec8ea417b4.png)

and here is result:
![image](https://user-images.githubusercontent.com/6390413/226424459-67c53f05-5dae-4093-aab7-ab9a70ede19d.png)

For the other models I got back the images I loaded (openpose, normal, canny), but not here. 
Why does it process the depthmap when the preprocessor is set to none? (Look at Log "using mask as input" line!)

### Steps to reproduce the problem

1. Load a depth map what I attached
2. Select none to Preprocessor
3. Set Model to control_depth
4. Generate image
![depth](https://user-images.githubusercontent.com/6390413/226425451-08a72913-78ad-4642-b5e5-d1e9938aae70.png)

### What should have happened?

The depth map should not be processed if I set it to none.

### Commit where the problem happens

webui: [a9fed7c3](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/a9fed7c364061ae6efb37f797b6b522cb3cf7aa2)
controlnet: [dbdd6b1](https://github.com/Mikubill/sd-webui-controlnet/commit/dbdd6b1b027b8dd02c1f13310f82cf359c617dc6)

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--xformers --ckpt-dir V:\!SDModels --ui-config-file 1my/ui-config-my.json --ui-settings-file 1my/config-my.json --autolaunch --gradio-img2img-tool color-sketch --skip-version-check
```

### Console logs

```Shell
Loading model from cache: control_openpose-fp16 [9ca67cc5]:21<00:00,  3.28it/s]
Loading preprocessor: none
Loading model: control_depth-fp16 [400750f6]
Loaded state_dict from [H:\Stable-Diffusion-Automatic\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_depth-fp16.safetensors]
ControlNet model control_depth-fp16 [400750f6] loaded.
using mask as input
Loading preprocessor: none
Loading model: control_normal-fp16 [63f96f7c]
Loaded state_dict from [H:\Stable-Diffusion-Automatic\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_normal-fp16.safetensors]
ControlNet model control_normal-fp16 [63f96f7c] loaded.
Loading preprocessor: none
Loading model: control_canny-fp16 [e3fe7712]
Loaded state_dict from [H:\Stable-Diffusion-Automatic\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_canny-fp16.safetensors]
ControlNet model control_canny-fp16 [e3fe7712] loaded.
Loading preprocessor: none
100%|██████████████████████████████████████████| 50/50 [00:19<00:00,  2.54it/s]
Total progress: 100%|██████████████████████████| 50/50 [00:24<00:00,  2.06it/s]
Total progress: 100%|██████████████████████████| 50/50 [00:24<00:00,  2.32it/s]
```
